cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -269,6 +276,17 @@ func IsDeletionFailedBefore(cVR *apis.CStorVolumeReplica) bool {
 	return false
 }
 
+// IsStatusOnline is to check if the status of cStorVolumeReplica object is
```

Comment:
instead of having so many functions which checks for each state, how about a single function that checks for given state, and returns true if matches or returns false if doesn't, like,
`func IsStatus(cVR *apis.CStorVolumeReplica, string status) bool`

Reply:
Okay @vishnuitta It is good to do in a single function.